src/main/java/com/ryszardpanda/medicalClinic/controller/DoctorController.java

```diff
@@ -51,10 +50,8 @@ public void updatePassword(@PathVariable String email, @RequestBody ChangePasswo
         doctorMapper.doctorToChangePasswordDTO(doctorService.updatePassword(email, updatedPassword));
     }
 
-    @PatchMapping("/doctors/{doctorId}/assign/{institutionId}")
+    @PatchMapping("/doctors/{doctorId}/assignToInstitution/{institutionId}")
```

Comment:
W przypadku sciezek nie uzywamy camel case. Standardem jest pisanie z "-".

"/doctors/{doctorId}/assign-to-institution/{institutionId}"

Reply:
ok